actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -15,13 +15,15 @@ def initialize
         @connection_class  = ApplicationCable::Connection
         @worker_pool_size  = 100
 
-        @channels_path = Rails.root.join('app/channels')
+        @channels_path = [Rails.root.join('app/channels')]
 
         @disable_request_forgery_protection = false
       end
 
       def channel_paths
-        @channels ||= Dir["#{channels_path}/**/*_channel.rb"]
+        @channels ||= channels_path.collect do |channel_path|
+          Dir["#{channel_path}/**/*_channel.rb"]
+        end.flatten
```

Comment:
While we're here we should change the instance variable to match the method name.

Also having a `channels_path` as an array feels off, when we have `channel_paths`. Don't see an entirely good direction here, but perhaps renaming `channels_path` to `channels_paths` could do?